### PR TITLE
Bump bottlerocket OVA version to latest

### PR DESCRIPTION
**Pull request by abhay-krishna**
Bump Bottlerocket version to their latest release.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/kubernetes-sigs/image-builder/BOTTLEROCKET_ADMIN_CONTAINER_METADATA [hunk 1]
```diff
@@ -1,2 +1,2 @@
-tag: v0.7.4
-imageDigest: sha256:ce84dd6e6280f8bb40ca8917304ce0526187c7f66c81017020d3503c7a9f821e
\ No newline at end of file
+tag: v0.8.0
+imageDigest: sha256:279ff0b939c8ebfae8fb5086751de831edee4c1ef307b6f0a27b553b1c2c9b52
```
[the remark this answers is not in the document]
Comment (vignesh-goutham) [reply projects/kubernetes-sigs/image-builder/BOTTLEROCKET_ADMIN_CONTAINER_METADATA:2]: yep

[the remark this answers is not in the document]
Comment (abhay-krishna) [reply projects/kubernetes-sigs/image-builder/BOTTLEROCKET_ADMIN_CONTAINER_METADATA:2]: it does

**Pull request merged**